### ISSUE TITLE
Cannot find symbol: class DOTA_COMBATLOG_TYPES

### ISSUE THREAD

**Issue by drsolaris** [skadistats/clarity-examples#6]
Hi! I have barely any Java experience but wanted to try out the examples here. I'm running into the following problem when trying to build any of the examples.

mvn -P info package

`[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project clarity-examples: Compilation failure
[ERROR] /Users/drsolaris/code/clarity-examples/src/main/java/skadistats/clarity/examples/combatlog/Main.java:[13,57] cannot find symbol
[ERROR] symbol:   class DOTA_COMBATLOG_TYPES
[ERROR] location: class skadistats.clarity.wire.s1.proto.DotaUsermessages`

Apologies if this is something really simple. Thanks for the great work.

**Comment by spheenik** [comment]
Hi!

Apologies: State of HEAD is bad atm, I am working on Source 2 , and 
moving around a lot of stuff.

DOTA_COMBAT_LOG_TYPES has been moved, and is now in:

skadistats.clarity.wire.common.proto.DotaUsermessagesCommon.DOTA_COMBATLOG_TYPES

I pushed a missing commit

https://github.com/skadistats/clarity-examples/commit/def6f92a8a20319763aaadc9cf9e9b2282086353

and new snapshots of clarity and clarity-protobuf, so, after upgrading 
your examples to the latest HEAD, it should work again.
